validator/client/validator.go

```diff
@@ -100,6 +101,15 @@ func (v *validator) WaitForActivation(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("could not receive validator activation from stream: %v", err)
 		}
+		if len(res.ValidatorEstimations) > 0 {
+			log.Info("Time till Activation")
+			for _, e := range res.ValidatorEstimations {
+				log.WithFields(logrus.Fields{
+					"pubkey":                fmt.Sprintf("%#x", e.PublicKey),
+					"Slots till Activation": fmt.Sprintf("%d", e.SlotsTillActivation),
```

Comment:
change to a consistent key names ‎capitalization

Reply:
"pubKey"
"timeTillActivation"

and use the time package to display the appropriate measure instead of just showing how many slots till activation